package.json

```diff
@@ -96,6 +96,7 @@
     "doctrine": "1.5.0",
     "eslint-import-resolver-node": "^0.3.2",
     "eslint-module-utils": "^2.4.1",
+    "find-up": "^2.0.0",
```

Comment:
i would prefer not to depend on any modules from this author; they drop support for old nodes too aggressively.

Reply:
**find-up** is a dependency of **read-pkg-up**, so already present in nested deps. For performances I only need to get the package.json path, not to load its content, That's why I don't use **read-pkg-up** directly.
If needed I can use full local code instead of adding a dependency

Reply:
ah, true, fair enough. i'll worry about excising those deps later.